README.md

```diff
@@ -91,11 +90,52 @@ Example projects on JVM are available for [Gradle](examples/example-jvm/build.gr
 
 ### Gradle
 
-You have to add the serialization plugin as the other [compiler plugins](https://kotlinlang.org/docs/reference/compiler-plugins.html):
+#### Using the `plugins` block
+
+You can setup the serialization plugin with the Kotlin plugin using [Gradle plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block):
+
+Kotlin DSL:
+
+```kotlin
+plugins {
+    kotlin("multiplatform") // or kotlin("jvm") or any other kotlin plugin
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.3.61"
```

Comment:
I haven't tried it myself, but does it work just as `kotlin("serialization")`? (judging by the following 'old way' example in Kotlin DSL)

Reply:
Actually, looking at the source code of the `kotlin()` extension function I could reduce it:
```
fun PluginDependenciesSpec.kotlin(module: String): PluginDependencySpec =
    id("org.jetbrains.kotlin.$module")
```
Good hint, thanks 👍